tests/unit-tests/test-class-quiz.php

```diff
@@ -876,6 +907,35 @@ public function testGetUserQuestionGrade() {
 
 	}//end testGetUserQuestionGrade()
 
+	/**
+	 * Testing $woothemes->quiz->get_user_question_grade without user_id argument.
+	 */
+	public function testGetUserQuestionGradeWithoutUserId() {
+		// setup the data needed for the assertions in this test
+		$test_user_id           = wp_create_user( 'testGetUserQuestionGrade', 'testGetUserQuestionGrade', 'testGetUserQuestionGrade@test.com' );
+		$test_lesson_id         = $this->factory->get_random_lesson_id();
+		$test_quiz_id           = Sensei()->lesson->lesson_quizzes( $test_lesson_id );
+		$test_user_quiz_answers = $this->factory->generate_user_quiz_answers( $test_quiz_id );
+		$test_user_grades       = $this->factory->generate_user_quiz_grades( $test_user_quiz_answers );
+		Sensei()->quiz->set_user_grades( $test_user_grades, $test_lesson_id, $test_user_id );
+		$test_question_id = array_rand( $test_user_grades );
+
+		$this->assertFalse(
+			Sensei()->quiz->get_user_question_grade( $test_lesson_id, $test_question_id ),
+			'Should return false when has no user_id argument and is not logged in'
+		);
+
+		wp_set_current_user( $test_user_id );
+		$retrieved_grade = Sensei()->quiz->get_user_question_grade( $test_lesson_id, $test_question_id );
+
+		// test if the the question grade can be retrieved
+		$this->assertEquals(
+			$test_user_grades[ $test_question_id ],
+			$retrieved_grade,
+			'Should return the correct grade if has no user_id argument but user is logged in'
+		);
+	}//end testGetUserQuestionGradeWithoutUserId()
```

Comment:
Same here.

Reply:
I took the opportunity to update all comment standards in this file: https://github.com/Automattic/sensei/pull/3016/commits/a3c6d6d41a6f34633c21ebfdb0618dfb809c56f0